src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Day/DiaryDayNotesInfoViewController.swift

```diff
@@ -0,0 +1,93 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+
+class DiaryDayNotesInfoViewController: DynamicTableViewController, DismissHandling {
+
+	// MARK: - Attributes
+
+	var finished: (() -> Void)?
+
```

Comment:
We don't have a MARK for attributes.
If it's required for the DeltaOnboardingViewControllerProtocol please move it to a dedicated section

Reply:
Thanks. I corrected it. It was not used at all.